packages/oidc-js/src/client.ts

```diff
@@ -67,6 +67,7 @@ const DefaultConfig = {
     enablePKCE: true,
     responseMode: null,
     scope: [OIDC_SCOPE],
+    sendCookiesInRequest: false,
```

Comment:
Shall we add an `s` at the end. WDYT?
```suggestion
    sendCookiesInRequests: false,
```

Reply:
Addressed with https://github.com/asgardeo/asgardeo-auth-js-sdk/pull/131/commits/118479173a48e08065a5c9ac6a57d4400cea248e